### PR TITLE
chore: community split v2

### PR DESCRIPTION
**Pull request by t3dotgg**
## TODO:
- [x] Fix versioning
- [x] Break out examples and non-core packages into separate repo
- [x] Fix linting and build
- [x] Create /sandbox for local examples when working
- [ ] Confirm with team this is all ready to go

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1.bug-report-pkgs.yml [hunk 1]
```diff
@@ -29,8 +29,8 @@ body:
...
     attributes:
       label: Link to reproduction
-      description: Please provide a link to a reproduction of the bug. This can be a link to a repository or a StackBlitz/CodeSandbox etc. Feel free to use our minimal reproduction template https://stackblitz.com/github/pingdotgg/uploadthing/tree/main/examples/minimal-appdir. You can also clone down any of our examples and use that as a starting point. You can do so by using `degit`, e.g. `npx degit pingdotgg/uploadthing/examples/minimal-appdir`
-      placeholder: https://stackblitz.com/github/pingdotgg/uploadthing/tree/main/examples/minimal-appdir
+      description: Please provide a link to a reproduction of the bug. This can be a link to a repository or a StackBlitz/CodeSandbox etc. Feel free to use our minimal reproduction template https://stackblitz.com/github/pingdotgg/uploadthing-community/tree/main/examples/minimal-appdir. You can also clone down any of our examples and use that as a starting point. You can do so by using `degit`, e.g. `npx degit pingdotgg/uploadthing/examples/minimal-appdir`
+      placeholder: https://stackblitz.com/github/pingdotgg/uploadthing-community/tree/main/examples/minimal-appdir
```
Comment (juliusmarminge) [inline .github/ISSUE_TEMPLATE/1.bug-report-pkgs.yml:33]: Should this link to sandbox or is that meant to be more of a kitchen sink?

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-18.18
+18.18
```
Comment (juliusmarminge) [inline .nvmrc:1]: fmt:)

File: examples/backend-adapters/README.md
Comment (juliusmarminge) [inline examples/backend-adapters/README.md:1]: I vote to keep this, I frequently use it to test stuff works for the different adapters

File: sandbox/minimal-appdir/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Minimal Next.js App Router example for UploadThing
 
-<a href="https://stackblitz.com/github/pingdotgg/uploadthing/tree/main/examples/minimal-appdir">
+<a href="https://stackblitz.com/github/pingdotgg/uploadthing-community/tree/main/examples/minimal-appdir">
```
Comment (juliusmarminge) [inline sandbox/minimal-appdir/README.md:3]: Wrong link?

File: examples/backend-adapters/README.md
Comment (markflorkowski) [reply examples/backend-adapters/README.md:1]: +1

File: docs/package.json [hunk 4]
```diff
@@ -11,13 +11,13 @@
     "debug": "NODE_OPTIONS='--inspect' next dev"
   },
   "dependencies": {
-    "@uploadthing/react": "6.0.2",
+    "@uploadthing/react": "^6.0.2",
```
Comment (markflorkowski) [inline docs/package.json:14]: Why? Also won't this be bumped automatically on each update?

**Review by markflorkowski — APPROVED** [review]

File: sandbox/minimal-appdir/README.md (hunk 3 above)
Comment (markflorkowski) [reply sandbox/minimal-appdir/README.md:3]: looks correct to me? Repo is just private rn

**Pull request closed without merging**